lib/tmuxinator/tmux_version.rb

```diff
@@ -0,0 +1,29 @@
+module Tmuxinator
+  module TmuxVersion
```

Comment:
Perhaps this module name could be more intention revealing? (e.g. SupportedTmuxVersions)